### DISCUSSION TITLE
求教，这是什么

### DISCUSSION THREAD

**Discussion by sunjx97** [qd-today/qd#205] [Q&A]
![image](https://user-images.githubusercontent.com/42996565/148761264-5b2de57a-a5af-4c9f-8bfb-d45ed84ad000.png)

就是这个，绿绿的东西

**Comment by a76yyyy** [comment]
那个是你提取的har里请求过多被自动分页了，是浏览器自动保存在har里的

**Reply by sunjx97** [reply]
有这东西存在老是把我的请求顺序搞乱，能搞个功能自动去除吗orz

**Reply by a76yyyy** [reply]
临时解决方案
可以先用文本编辑软件比如vscode打开har文件，右键格式化文档后，把无用的信息删除后保存再导入框架
